client/lib/wp/sync-handler/test/cache-index.js

```diff
@@ -30,18 +30,13 @@ describe( 'cache-index', () => {
 	describe( '#getAll', () => {
 		beforeEach( clearLocal );
 
-		it( 'should return undefined for empty localforage', ( done ) => {
+		it( 'should return undefined for empty localforage', () => {
 			return cacheIndex.getAll()
 				.then( res => {
-					try {
```

Comment:
FYI: Mocha wants us to return `Promise` or use `done` with new library versions, we can't use both :)